stub-idp/src/integration-test/java/uk/gov/ida/apprule/steps/Cookies.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.ida.apprule.steps;
+
+import uk.gov.ida.stub.idp.cookies.CookieNames;
+import uk.gov.ida.stub.idp.cookies.HttpOnlyNewCookie;
+
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Cookies {
+    Map<String, NewCookie> cookies;
+
+    public Cookies() {
+        cookies = new HashMap<>();
+    }
+
+    public void extractCookies(Response response) {
+        response.getCookies().forEach((k,v) -> this.cookies.put(k,v));
+    }
+
+    public void setSessionCookie(String sessionId) {
+        this.cookies.put(CookieNames.SESSION_COOKIE_NAME,
+                new HttpOnlyNewCookie(CookieNames.SESSION_COOKIE_NAME,
+                                      sessionId,
+                                "/",
+                             "",
```

Comment:
Minor/pedantic thing ... can we fix the indentation here?

Reply:
Done. I fell foul of IntelliJ inserting the parameter names in front of literals.